### PR TITLE
refactor: Move aside to its own component in Home view.

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-6525

Move aside to its own component in Home view, to reduce the size of the Home view.

### CODE REVIEW

File: packages/x-components/src/views/home/aside.vue [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            <ChevronDown />
+          </template>
+          <!-- Filters -->
+          <SlicedFilters max="4" :filters="facet.filters">
+            <FiltersList v-slot="{ filter }">
+              <HierarchicalFilter :filter="filter" :data-test="`${facet.label}-filter`" />
+            </FiltersList>
+          </SlicedFilters>
+        </BaseHeaderTogglePanel>
+      </template>
```
Comment (javieri-empathy) [inline packages/x-components/src/views/home/aside.vue:45]: I guess this can be now reduced to one facet using the slot by model name.

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: packages/x-components/src/views/home/Home.vue [hunk 2]
```diff
@@ -629,7 +462,9 @@
     protected sortDropdownAnimation = CollapseHeight;
     protected selectedColumns = 4;
     protected sortValues = ['', 'price asc', 'price desc'];
-    protected controls = {
+
+    @XProvide('controls')
+    public controls = {
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply packages/x-components/src/views/home/Home.vue]: > done

File: packages/x-components/src/views/home/aside.vue [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    <FacetsProvider :facets="staticFacets" />
+    <ClearFilters />
+    <SelectedFiltersList>
+      <template #default="{ filter }">
+        <SimpleFilter :filter="filter" class="x-tag" />
+      </template>
+    </SelectedFiltersList>
+
+    <!-- Facets -->
+    <Facets class="x-list--gap-06" renderable-facets="!rootCategories_facet">
```
[the remark this answers is not in the document]
Comment (tajespasarela) [reply packages/x-components/src/views/home/aside.vue]: done

File: packages/x-components/src/views/home/aside.vue (hunk 1 above)
Comment (tajespasarela) [reply packages/x-components/src/views/home/aside.vue]: done

**Review by javieri-empathy — APPROVED** [review]

**Pull request merged**